boot.php

```diff
@@ -1,13 +1,18 @@
 <?php
 	if (!rex::isBackend()) {
 		rex_extension::register('RESPONSE_SHUTDOWN', function (rex_extension_point $ep) {
+            $Addon = rex_addon::get('errormail');
```

Comment:
vermutlich kannst du hier mit `$this` arbeiten

Reply:
du hast recht, danke